Makefile

```diff
@@ -32,11 +32,17 @@ ifeq ($(PYTHON),)
 endif
 PYTHON_BIN := $(shell which $(PYTHON))
 PYTHON_VER := $(word 2,$(shell $(PYTHON) -V 2>&1))
+GO_MIN_VER := 1.5
+GO_VER := $(subst go,, $(word 3,$(shell go version 2>&1)) )
 
 ifeq ($(filter 2.7.%,$(PYTHON_VER)),)
   $(error unsupported Python version $(PYTHON_VER), Grumpy only supports 2.7.x. To use a different python binary such as python2, run: 'make PYTHON=python2 ...')
 endif
 
+ifneq "$(GO_MIN_VER)" "$(word 1, $(sort $(GO_MIN_VER) $(GO_VER)))"
```

Comment:
If we can assume that sed is going to be present then yes, that approach is more readable.
Otherwise the proposed approach only depends on make.
To check for major, minor we can do something like.

`
GO_VER := $(subst go,, $(word 3,$(shell go version 2>&1)) )
GO_MAJ_VER := $(word 1,$(subst ., ,$(GO_VER) ))
GO_MIN_VER := $(word 2,$(subst ., ,$(GO_VER) ))`

and then compare accordingly.